roles/chaosd/tasks/setup_nginx_proxy.yaml

```diff
@@ -0,0 +1,49 @@
+- name: Create chaosd nginx location configuration file
+  ansible.builtin.copy:
+    content: |
+      # Chaosd chaos engineering API endpoint
```

Comment:
can you guys modify this section to reuse the nginx we have setup on all hosts anyway? Most CLs already expose their RPCs by using `VIRTUAL_HOST`, `VIRTUAL_PORT` and `LETSENCRYPT_HOST`. Is that enough for chaosd? if not, what else would be needed and can we achieve it without having to add nginx to the chaod role?